src/managed/CommonManaged.props

```diff
@@ -5,7 +5,24 @@
 
   <PropertyGroup>
```

Comment:
Yea, sort of.  Right now we only have the 2 projects, but if we added more SDK-based projects, they would want most of this stuff.

We do have 3 "test" projects that are SDK-based, but they don't import this props file.

Hopefully, we can get more "common" properties refactored into smaller .props files, so they can be shared and we can keep duplication to a minimum.